lib/_emerge/Package.py

```diff
@@ -314,6 +314,10 @@ def _validate_deps(self):
         eapi = self.eapi
         dep_eapi = eapi
         dep_valid_flag = self.iuse.is_valid_flag
+
+        if not force and not self.root_config.settings.repositories[self.repo].volatile:
```

Comment:
Need to address @zmedico's comment at https://github.com/gentoo/portage/pull/961#discussion_r1051689888.